client/component/comment.js

```diff
@@ -213,14 +215,23 @@ function EditComment({comment, big=false, setComment, topLevel, onEditingDone, o
             onEditingDone(comment);
         }
     }
+    const {isMobile} = getDeviceInfo();
```

Comment:
Can we split a getIsMobile function out of getDeviceInfo? (probably memoized, since devices don't change between being mobile and not)

getDeviceInfo is intended to gather a load of info for logging, and could potentially be expensive in the future (though it isn't now). 

This particular component renders every keypress, so it's super important for it to be fast.

Reply:
done